### ISSUE TITLE
Program type already present: androidx.test.uiautomator.AccessibilityNodeInfoHelper

### ISSUE THREAD

**Issue by nikoskyr** [SchibstedSpain/Barista#268]
Hello,

I tried to integrate Barista to my Android Espresso testing scope, but I got the following error:

`AGPBI: {"kind":"error","text":"Program type already present: androidx.test.uiautomator.AccessibilityNodeInfoHelper","sources":[{}],"tool":"D8"} Task :app:buildInfoGeneratorDebug FAILURE: Build failed with an exception. * What went wrong: Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'. com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: Learn how to resolve the issue at https://developer.android.com/studio/build/dependencies#duplicate_classes. Program type already present: androidx.test.uiautomator.AccessibilityNodeInfoHelper`

Project is updated to androidx and the used sdk is 28.

Is it a known issue? Could you please check it? 

Thanks in advance!!

-Nikos-

**Comment by lachudra87** [comment]
barista use `uiautomator` lib in v18 version and for androidx you probably use non v18 version. Please check if below will fix your issue:
```
    androidTestImplementation ('com.schibsted.spain:barista:2.7.0') {
        exclude group: 'com.android.support.test.uiautomator', module: 'uiautomator-v18'
    }
```

**Comment by Sloy** [comment]
Hi @nikoskyr, sorry for the late response. I didn't get my hands in AndroidX until recently.

Yes, there seems to be an issue when using Barista and `androidx.test.uiautomator:uiautomator:2.2.0`.    The workaround from @lachudra87 does the trick. I think that is the best approach until we do https://github.com/SchibstedSpain/Barista/issues/262

**Comment by Sloy** [comment]
Correction, there's no other artifact, it's just a wrong name in the jetifier 🤦‍♂️ 
I guess the only option is @lachudra87's workaround unless Google fixes the issue on their side.

**Comment by Sloy** [comment]
Should be fixed with the AndroidX migration in Barista 3.0.0 😃
